src/dist.rs

```diff
@@ -0,0 +1,308 @@
+use crate::{default_build_command, default_dist_dir, metadata};
+use anyhow::{ensure, Context, Result};
+use clap::Parser;
+use std::{fs, path::PathBuf, process};
+use wasm_bindgen_cli_support::Bindgen;
+
+/// An helper to generate the distributed package
```

Comment:
Spellchecking :sweat: 
```suggestion
/// A helper to generate the distributed package
```